src/pykeen/losses.py

```diff
@@ -530,6 +541,12 @@ def forward(
             neg_scores - pos_scores + self.margin,
         ))
 
+    def get_config(self) -> Mapping[str, Any]:  # noqa: D102
+        config = dict(super().get_config())
```

Comment:
@cthoyt this first line is the same for all subclasses - any idea how to elegantly get rid of this duplication (while still maintaining readability :sweat_smile: )?

Reply:
not while maintaining readability :p